odk1-src/getting-started.rst

```diff
@@ -1,90 +1,88 @@
 Getting Started With ODK
 =========================
 
-.. _using-odk:
+This document walks you through a very basic setup process,
+to get you familiar with using Open Data Kit.
 
-Using ODK
------------
+You will:
 
-In most cases, users of ODK:
+.. contents::
+ :local:
 
-- Create survey forms using `Build <https://build.opendatakit.org/>`_ or `XLSForm <http://xlsform.org/>`_
-- Upload forms to an Aggregate server
-- Load forms into Collect on an Android device
-- Use Collect to fill out forms with :term:`participants <participant>`
-- Upload survey data from Collect to Aggregate
-- Analyze or export data in Aggregate
+.. _getting-started-install-collect:
 
-This requires:
-
-- :doc:`Installing Collect on a phone or other mobile device <collect-install>`
-- :ref:`Installing Aggregate on a server <installing-aggregate>`
-
-.. tip::
+Install Collect
+---------------------
 
-  While this is the *typical* use pattern, it is not the only way to do things. For example:
+The easiest way to install the Collect App is `to get it from the Google Play store <https://play.google.com/store/apps/details?id=org.odk.collect.android&hl=en>`_.
 
-  - Forms can be created using tools other than Build or XLSForm
-  - Forms can be transfered directly to Collect, without using Aggregate
+.. seealso:: :doc:`collect-install`
 
-  ODK is a very flexible set of tools, and organizations will find their own best practices for adopting it.
+.. _getting-started-install-aggregate:
 
-.. _install-collect:
+Install Aggregate (optional)
+------------------------------
 
-Install Collect
----------------------
+The easiest way to set up Aggregate is to
+:doc:`install it on Google App engine <aggregate-app-engine>`. 
 
-The easiest way to install the ODK Collect App is `to get it from the Google Play store <https://play.google.com/store/apps/details?id=org.odk.collect.android&hl=en>`_.
+You'll set up a new Google Cloud project, and then run the install utility locally. This will connect to your Google Cloud account and install Aggregate there.
 
-For other installation options, see :doc:`collect-install`.
+Alternatively, if you just want to try things out,
+you can use the `Aggregate sandbox server`_.
 
-.. _installing-aggregate:
+.. _Aggregate sandbox server: https://sandbox.aggregate.opendatakit.org:8443
```

Comment:
We should add a warning that `This is a demonstration server. All forms and data on this server are public and are occasionally deleted.`